tools/findCircularDepend.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/python
```

Comment:
Hi Steve, 

Can you add the standard python script interface from the Tools directory.    Also should we have a new sub directory instead of putting it directly in tools?

Adding code similar to this:
```def main(arg1, arg2, etc):
    # do whatever the script does


if __name__ == "__main__":
    main(sys.argv[1], sys.argv[2], sys.argv[3])
```
will allow this script to be called as a subroutine from other scripts, we could then pass an option to case.build to run it from a case.